packages/adapters/ethers/src/client.ts

```diff
@@ -631,9 +631,12 @@ export class EthersAdapter {
       if (provider) {
         const { addresses, chainId } = await EthersHelpersUtil.getUserInfo(provider)
```

Comment:
why source the infor from here ?

Reply:
I think it's because during the construct of the adapter we'd need to get the address and chain id directly from the provider so users are still connected even if they refresh the page. The logic happens [here](https://github.com/reown-com/appkit/blob/main/packages/adapters/ethers/src/client.ts#L522-L524).